client/components/tinymce/plugins/simple-payments/dialog/form.jsx

```diff
@@ -82,18 +82,18 @@ const validate = ( values, props ) => {
 	const errors = {};
 
 	if ( ! values.title ) {
-		errors.title = translate( 'This can not be empty.' );
+		errors.title = translate( 'This cannot be empty.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **70** times:
`translate( 'Passwords cannot be empty.' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/error-message-simple-payments)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).